seeds/01_items.js

```diff
@@ -0,0 +1,27 @@
+var faker = require('faker');
+var sprintf = require('sprintf-js').sprintf;
+
+let createRecord = (knex, id) => {
+  let imageName = sprintf('%05s.jpg', id)
```

Comment:
is this for generating an img?

Reply:
this is to create the "image name". The `"%05s.jpg", id` will make the total length of numbers before jpg be 5 so it will put the id right before jpg and then 0 pad that with however many 0s needed to get to a length of 5